src/Agent.Worker/TestResults/Legacy/TestResultsServer.cs

```diff
@@ -102,6 +107,26 @@ public void InitializeServer(VssConnection connection, IExecutionContext executi
             return await TestHttpClient.CreateTestSubResultAttachmentAsync(reqModel, projectName, testRunId, testCaseResultId, testSubResultId, cancellationToken);
         }
 
+        public async Task<TestResultsSettings> GetTestResultsSettingsAsync(string project, TestResultsSettingsType? settingsType = null, object userState = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return await TestHttpClient.GetTestResultsSettingsAsync(project, settingsType);
```

Comment:
What's the type of  `TestHttpClient` ?

Reply:
It's actually a base class and it can be `TestResultsHttpClient` or `TestManagementHttpClient`. 
And your API is only there in `TestResultsHttpClient`, so please check what's the type of it.